core/src/main/java/curso/usecase/CreateCourseUS.java

```diff
@@ -0,0 +1,26 @@
+package curso.usecase;
+
+import curso.input.Persistencia;
+import curso.modelo.Course;
+import curso.modelo.level;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class CreateCourseUS implements InterfaceCreateCourseUS {
```

Comment:
En este caso, si la abreviatura US representa que es un usecase, usar UC por ejemplo, asi no se presta a confusión al leer.
```suggestion
public class CreateCourseUC implements InterfaceCreateCourseUC {
```